modules/sdk/src/main/scala/org/tessellation/sdk/domain/trust/storage/OrdinalTrustMap.scala

```diff
@@ -0,0 +1,36 @@
+package org.tessellation.sdk.domain.trust.storage
+
+import org.tessellation.schema.SnapshotOrdinal
+
+import derevo.cats.eqv
+import derevo.circe.magnolia.encoder
+import derevo.derive
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.numeric.NonNegLong
+
+@derive(eqv, encoder)
+case class OrdinalTrustMap(
+  ordinal: SnapshotOrdinal,
+  peerLabels: PublicTrustMap,
+  trust: TrustMap
+)
+
+object OrdinalTrustMap {
+
+  val ordinalTrustUpdateInterval: NonNegLong = 1000L
+
+  val ordinalTrustUpdateDelay: NonNegLong = 500L
```

Comment:
can you please move it to the configuration?

Reply:
They been moved to

```
  case class TrustStorageConfig(
    ordinalTrustUpdateInterval: NonNegLong,
    ordinalTrustUpdateDelay: NonNegLong
  )
```

which is included in `SdkConfig`